<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetctlAutoWindow</class>
 <widget class="QMainWindow" name="NetctlAutoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="layout_main">
      <item>
       <layout class="QVBoxLayout" name="layout_available">
        <item>
         <widget class="QLabel" name="label_available">
          <property name="text">
           <string>Available profiles</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_available">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout_buttons">
        <item>
         <spacer name="spacer_buttonsUp">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_addAll">
          <property name="icon">
           <iconset theme="arrow-right-double">
            <normaloff/>
           </iconset>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_add">
          <property name="icon">
           <iconset theme="arrow-right">
            <normaloff/>
           </iconset>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_remove">
          <property name="icon">
           <iconset theme="arrow-left">
            <normaloff/>
           </iconset>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_removeAll">
          <property name="icon">
           <iconset theme="arrow-left-double">
            <normaloff/>
           </iconset>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer_buttonsDown">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout_active">
        <item>
         <widget class="QLabel" name="label_active">
          <property name="text">
           <string>Active profiles</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_active">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSwitch"/>
    <addaction name="actionEnable"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionEnable">
   <property name="icon">
    <iconset theme="edit-add">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Enable profile</string>
   </property>
  </action>
  <action name="actionSwitch">
   <property name="icon">
    <iconset theme="dialog-apply">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Switch to profile</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
